velox/functions/prestosql/IPAddressFunctions.h

```diff
@@ -16,11 +16,13 @@
 #pragma once
 
 #include "velox/functions/prestosql/types/IPAddressType.h"
```

Comment:
Maybe add some tests in FunctionRegistryTest or CustomTypeTest to ensure that we can register functions with type IPPREFIX and retrieve them ?

Reply:
Added simple function registration and resolution test into FunctionRegistryTest